plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -60,7 +59,7 @@ export class EventsProcessor {
         timestamp: DateTime,
         eventUuid: string,
         person: IngestionPersonData | undefined = undefined
-    ): Promise<PreIngestionEvent | null> {
+    ): Promise<IngestionEvent | null> {
```

Comment:
Note that this makes this a (confusing) semantic change - previously a pre ingestion event preceded ingestion, now "ingestionEvent" can also mean an event that's not yet ready for ingestion.

Reply:
Well, `PreIngestionEvent` and `IngestionEvent` were one and the same. I can see how in some future that could perhaps maybe change, but right now the distinction was only confusing.